src/transliteration/ui/WordInfoAlignments.tsx

```diff
@@ -53,16 +68,27 @@ const AlignedTokens = withData<
                       &nbsp;
                     </Col>
                     <Col>
-                      <LemmaInfo
-                        word={lineToken.token}
-                        dictionary={dictionary}
-                        manuscriptLines={manuscripts}
-                      />
+                      <span className="word-info__variant--guideword">
+                        <span>{lineToken.token.variant?.value}</span>
+                        {isDifferentLemma && (
```

Comment:
If the lemma is different, show "_lemma_; link_to_lemma", else just show "_lemma_".